<?xml version="1.0"?>
<launch>

  <node name="objectrecognition" pkg="objectrecognition"
      type="horizontal_crop"
      output="screen" >

    <param name="radius" value= ".0065"/>
    <param name="thereshold" value= "1" />
    <param name="resolution" value= ".02" />


    <param name="orientation_x" value="0.707068" />
    <param name="orientation_y" value="0.0" />
    <param name="orientation_z" value="0.0" />
    <param name="orientation_w" value="-0.707068" />


  </node>
</launch>